src/ansys/dpf/post/result_workflows/_build_workflow.py

```diff
@@ -110,12 +111,18 @@ def _requires_manual_averaging(
         return True
     if category == ResultCategory.equivalent and base_name[0] == "E":  # strain eqv
         return True
```

Comment:
Moved requires_manual_averaging in one place, because it is confusing if one part of the decision is done in the selection and the other part is done in the free function.